build-helpers/glib-tap.mk

```diff
@@ -68,18 +68,18 @@ check_DATA =
 #   dist_installed_test_scripts, dist_installed_test_data, dist_installed_test_extra_scripts
 #   dist_uninstalled_test_scripts, dist_uninstalled_test_data, dist_uninstalled_test_extra_scripts
 #
-# Note that no file is automatically disted unless it appears in one of the dist_ variables.  This follows the
-# standard automake convention of not disting programs scripts or data by default.
+# Note that no file is automatically distributed unless it appears in one of the dist_ variables.  This follows the
+# standard automake convention of not distributing programs scripts or data by default.
 #
-# test_programs, test_scripts, uninstalled_test_programs and uninstalled_test_scripts (as well as their disted
+# test_programs, test_scripts, uninstalled_test_programs and uninstalled_test_scripts (as well as their distributed
```

Comment:
Happy to drop this (or any other file)

Reply:
The files in `build-helpers` are from glib-2.58 (and were [dropped](https://gitlab.gnome.org/GNOME/glib/-/commit/b3efef5b6ff2515551f300aed7ca16bd728e836a) in later glib releases; although the claim to continue to support autotools-using projects, they still dropped those test helpers...).
I'd still like to keep them as close as possible to the original upstream version.